### ISSUE TITLE
upload photo to vendor receipt and check request

### ISSUE THREAD

**Issue by babybunny** [babybunny/rebuildingtogethercaptain#304]
basically any of the expenses should support an image attachment.

**Comment by jophillipsrtp** [comment]
This is a priority issue for me. How do I change this to High-Priority?

**Comment by babybunny** [comment]
Do you see labels to the right of the issue with the little gear next to
it. If you click the gear you should be able to add the high priority
label. Try it!

**Comment by jophillipsrtp** [comment]
Yes, I tried the gear box and when I click on it nothing happens- Do I need to be an Administrator to make that change? Greg is also not able to use the gear, but Cari is?? THANK YOU!!!

**Comment by babybunny** [comment]
Created a new issue to sort this out.  How "meta" :\

https://github.com/babybunny/rebuildingtogethercaptain/issues/422

**Comment by babybunny** [comment]
very similar to issue #172.  actually probably exactly the same thing.

**Comment by jophillipsrtp** [comment]
This will mostly be used for Captains to automatically attach receipts (PDF usually) at the same time that they submit a check request, so if it could say something that captains would see like "Upload Receipt Here" that would be ideal!

**Comment by jophillipsrtp** [comment]
Hi guys, Just wondering how this issue is coming along? I have captains that are starting to submit check requests and if they can have the availability of submitting their scanned receipt along with the check request on ROOMs that would be great for everyone. Thank you!!! (this is higher priority than the site budgets at this point, FYI)

**Comment by andy-d** [comment]
Hey Jo!
I’m going to look at this today and or tomorrow. Fortunately the attachment
API that we created for the SOW feature is general enough that this one
should be pretty easy. Thanks for the ping on it!
Andy

**Comment by andy-d** [comment]
Considering a few designs for this in an attempt to balance lift with usability - more tomorrow ...
